scripts/common.py

```diff
@@ -579,6 +581,24 @@ def inplace_replace_file(filename, regex_pattern, replacement_string):
         raise
 
 
+def wait_for_port_to_be_open(port_number):
+    """Wait until the port is open.
+
+    Args:
+        port_number: int. The port number to wait.
+    """
+    waited_seconds = 0
+    while (not is_port_open(port_number)
+           and waited_seconds < MAX_WAIT_TIME_FOR_PORT_TO_OPEN_SECS):
+        time.sleep(1)
+        waited_seconds += 1
+    if (waited_seconds == MAX_WAIT_TIME_FOR_PORT_TO_OPEN_SECS
+            and not is_port_open(port_number)):
+        python_utils.PRINT('Failed to start server on port %s, exiting ...' %
```

Comment:
Break after `(`

Reply:
Just curious, is the linter not catching this?

Reply:
@vinitamurthi, We don't have a lint check for that as of now, we are working on implement it! 
![image](https://user-images.githubusercontent.com/16653571/87751399-86d50100-c81b-11ea-9095-105870905b9b.png)
([ref](https://github.com/oppia/oppia/issues/8423))

Reply:
Done.